src/pages/dataTableUtilities/getColumns.js

```diff
@@ -479,6 +480,14 @@ const COLUMNS = () => ({
 
   // DATE COLUMNS
 
+  [COLUMN_NAMES.DATE_OF_BIRTH]: {
+    type: COLUMN_TYPES.DATE,
+    key: COLUMN_KEYS.DATE_OF_BIRTH,
+    title: 'D.O.B',
```

Comment:
:nit ( perhaps even, '*very* nit-picky' ) `D.O.B.` really, as the Birth is also abbreviated

Reply:
Nice #1693